src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -161,6 +166,127 @@ public void testAddQuestionAction() throws Exception {
     }
 
     @Override
+    protected void testDestructiveChanges() {
+        // Create a dummy response
+        FeedbackQuestionAttributes question = BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1);
+        FeedbackResponseAttributes fra = new FeedbackResponseAttributes(
+                                                feedbackSessionName,
+                                                courseId,
+                                                question.getId(),
+                                                question.getQuestionType(),
+                                                "tmms.test@gmail.tmt",
+                                                null,
+                                                "alice.b.tmms@gmail.tmt",
+                                                null,
+                                                null);
+        BackDoor.createFeedbackResponse(fra);
+        feedbackEditPage.reloadPage();
+        feedbackEditPage.isAlertClassEnabledForVisibilityOptions(1);
+
+        // readying for tests
+        FeedbackConstantSumQuestionDetails csQuestion = (FeedbackConstantSumQuestionDetails) question.getQuestionDetails();
+        String prevVal = csQuestion.getConstSumOptions().get(0);
+
+        ______TS("CONST SUM: testing changes to const sum option");
+        // make changes to option title, must display modal
+        feedbackEditPage.clickEditQuestionButton(1);
+        feedbackEditPage.fillConstSumOption(0, "modified const sum option", 1);
+        feedbackEditPage.clickSaveExistingQuestionButton(1);
+        feedbackEditPage.waitForConfirmationModalAndClickCancel();
+
+        // revert changes, must not display modal
+        feedbackEditPage.fillConstSumOption(0, prevVal, 1);
+        feedbackEditPage.clickSaveExistingQuestionButton(1);
+        feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_EDITED);
+        feedbackEditPage.isAlertClassEnabledForVisibilityOptions(1);
+
+        ______TS("CONST SUM: testing adding a const sum option");
+        // add an option, must display modal
+        feedbackEditPage.reloadPage();
+        feedbackEditPage.clickEditQuestionButton(1);
+        feedbackEditPage.clickAddMoreConstSumOptionLink(1);
+        feedbackEditPage.fillConstSumOption(3, "new const sum option", 1);
+        feedbackEditPage.clickSaveExistingQuestionButton(1);
+        feedbackEditPage.waitForConfirmationModalAndClickCancel();
+
+        // revert changes, must not display modal
+        feedbackEditPage.clickRemoveConstSumOptionLink(3, 1);
+        feedbackEditPage.clickSaveExistingQuestionButton(1);
+        feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_QUESTION_EDITED);
+        feedbackEditPage.isAlertClassEnabledForVisibilityOptions(1);
+
+        ______TS("CONST SUM: testing deleting a const sum option");
+        // delete last option, must display modal
+        feedbackEditPage.reloadPage();
```

Comment:
remove reloadPage?

Reply:
Reloading page is necessary to clear the status message. But I guess the redundant ones can be removed. Will do.